bucket/winscp.json

```diff
@@ -13,7 +13,10 @@
         "    ) | Add-Content -Path \"$dir\\winscp.ini\" -Encoding ASCII -Force",
         "}"
     ],
-    "bin": "WinSCP.exe",
+    "bin": [
+        "WinSCP.exe",
+        "WinSCP.com"
```

Comment:
This will not work as `winscp.exe` will be immediatelly overriden with `winspc.com` Add just `.com` to bin.

Reply:
Hmm... May I suggest this instead?
```json
...
"pre_install": [
   ...
   "Set-Content \"$dir\\winscp-cli.cmd\" (@('@echo off', '\"%~dp0WinSCP.com\" %*') -join \"`r`n\") -Encoding ASCII"
],
"bin": [
    "WinSCP.exe",
    "winscp-cli.cmd"
],
...
```

We could use `winscp-cli` for commandline usage. (Tested and working)

Reply:
Why do you add `winscp.exe` in first?

Reply:
Backward compatibility?
I'm not sure to understand the question.
The current need is to have a way to also use the commandline version (`WinSCP.com`) along with the common GUI one (`WinSCP.exe`).

Reply:
Nonsense.

So you will prioritize GUI application as command line instead of proper command line application with proper command line argument handler.

Reply:
Totally agreed but I don't want to upset users that might be using it as it currently is...
This will shim only the `.com` version, no?
Should proceed anyway?

Reply:
This will shim exe file.
Then this exe will be overriden of com shim, which will lead to only com shimmed, which will result for erorrs on updates / installation.

Reply:
Ok, so `"bin": "WinSCP.com"` only and no backward compatibility, if anyone was using it this way?

Reply:
I am forbidding spawning GUI application from terminal in all cases, as it is completely irrelevant and nonsense, so this decision could not be done by me.

Reply:
Done.
You can assign the one who can take the final word/action.
Thanks!